mediathread/settings_shared.py

```diff
@@ -179,7 +179,7 @@ def no_reject(request, reason):
 
 ACCOUNT_ACTIVATION_DAYS = 7
 
-CORS_ORIGIN_ALLOW_ALL = True
+CORS_ALLOW_ALL_ORIGINS = True
```

Comment:
lol - was that really necessary?

Reply:
hah i know right. So, the old setting name is still working - this is just futureproofing it. This doesn't fix the Dartmouth error - I'm still looking into that, able to reproduce it.